src/main/java/setup/BaseTest.java

```diff
@@ -1,67 +1,166 @@
+    @BeforeClass(alwaysRun = true) //changed for running two tests in one suite
+    public void setUp(String platformName,
+                      String appType,
+                      @Optional("") String deviceName,
+                      @Optional("") String udid,
+                      @Optional("") String browserName,
+                      @Optional("") String app,
+                      @Optional("") String appPackage,
+                      @Optional("") String appActivity,
+                      @Optional("") String bundleId) throws Exception {
+        System.out.println("Before: app type - " + appType);
+        setAppiumDriver(platformName, deviceName, udid, browserName, app, appPackage, appActivity, bundleId);
         setPageObject(appType, appiumDriver);
-
     }
 
-    @AfterSuite(alwaysRun = true)
-    public void tearDown() throws Exception {
+    @AfterClass(alwaysRun = true) //changed for running two tests in one suite
+    public void tearDown() {
         System.out.println("After");
         appiumDriver.closeApp();
     }
 
-    private void setAppiumDriver(String platformName, String deviceName, String browserName, String app){
+    private void setAppiumDriver(String platformName, String deviceName, String udid, String browserName,
+                                 String app, String appPackage, String appActivity, String bundleId) {
         DesiredCapabilities capabilities = new DesiredCapabilities();
         //mandatory Android capabilities
-        capabilities.setCapability("platformName",platformName);
-        capabilities.setCapability("deviceName",deviceName);
+        capabilities.setCapability("platformName", platformName);
+        capabilities.setCapability("deviceName", deviceName);
+        capabilities.setCapability("udid", udid);
 
-        if(app.endsWith(".apk")) capabilities.setCapability("app", (new File(app)).getAbsolutePath());
+        if (app.endsWith(".apk")) {
+            capabilities.setCapability("app", (new File(app)).getAbsolutePath());
+        }
 
         capabilities.setCapability("browserName", browserName);
-        capabilities.setCapability("chromedriverDisableBuildCheck","true");
+        capabilities.setCapability("chromedriverDisableBuildCheck", "true");
+
+        // Capabilities for test of Android native app on EPAM Mobile Cloud
+        capabilities.setCapability("appPackage", appPackage);
+        capabilities.setCapability("appActivity", appActivity);
+
+        // Capabilities for test of iOS native app on EPAM Mobile Cloud
+        capabilities.setCapability("bundleId", bundleId);
+        //        if(platformName.equals("iOS")) capabilities.setCapability("automationName","XCUITest");
 
         try {
-            appiumDriver = new AppiumDriver(new URL(System.getProperty("ts.appium")), capabilities);
+
+            String url = "https://" + System.getenv("USERNAMEMOBIT") + ":" + URLEncoder.encode(System.getenv("TOKEN"))
+                + "@app.mobitru.com/wd/hub";
```

Comment:
It's better to use a template + format here instead of concatenation

Reply:
Fixed